data/db_config.js

```diff
@@ -1,8 +1,12 @@
 const knex = require('knex');
 
+
 const knexfile = require('../knexfile');
+require('dotenv').config();
 
 const env = process.env.NODE_ENV || 'development';
+
+// const env = 'development';
```

Comment:
Remove `require('dotenv').config();`

Add a comment above note for only using 'development' environment and it working for testing

Reply:
This has been completed with a note added